hudi-examples/README.md

```diff
@@ -19,27 +19,31 @@ This directory contains examples code that uses hudi.
 
 To run the demo: 
 
-  1. Configure your `SPARK_MASTER` env variable, yarn-cluster mode by default.
-  2. For hudi write client demo and hudi data source demo, just use spark-submit as common spark app
-  3. For hudi delta streamer demo of custom source, run `bin/custom-delta-streamer-example.sh`
-  4. For hudi delta streamer demo of dfs source:
+1. Configure your `SPARK_MASTER` env variable, yarn-cluster mode by default.
+2. For hudi write client demo and hudi data source demo, just use spark-submit as common spark app
+3. For hudi streamer demo of custom source, run `bin/custom-streamer-example.sh`
+4. For hudi streamer demo of dfs source:
 
-      4.1 Prepare dfs data, we have provided `src/main/resources/delta-streamer-config/dfs/source-file.json` for test
+   4.1 Prepare dfs data, we have provided `hudi-examples-spark/src/main/resources/-streamer-config/dfs/source-file.json`
+   for test
 
-      4.2 Run `bin/dfs-delta-streamer-example.sh`
+   4.2 Run `bin/dfs-streamer-example.sh`
 
-  5. For hudi delta streamer demo of dfs source:
+5. For hudi streamer demo of dfs source:
 
-      5.1 Start Kafka server
+   5.1 Start Kafka server
 
-      5.2 Configure your Kafka properties, we have provided `src/main/resources/delta-streamer-config/kafka/kafka-source.properties` for test
+   5.2 Configure your Kafka properties, we have
+   provided `hudi-examples-spark/src/main/resources/streamer-config/kafka/kafka-source.properties` for test
 
-      5.3 Run `bin/kafka-delta-streamer-example.sh`
+   5.3 Run `bin/kafka-streamer-example.sh`
 
-      5.4 Continuously write source data to the Kafka topic your configured with `hoodie.deltastreamer.source.kafka.topic` in `kafka-source.properties`
+   5.4 Continuously write source data to the Kafka topic your configured with `hoodie.deltastreamer.source.kafka.topic`
```

Comment:
Are there plans of renaming the config keys as well in a subsequent patch? We could do that in a backwards compatible manner using alias.

Reply:
I thought about that, i.e., renaming the deltastreamer config prefix `hoodie. delta streamer.` to `hoodie.streamer.`.  Since most of the deltastreamer configs are using `config.key()` directly, e.g., `props.getBoolean(config.key(), defaultValue)`, we need to chase such callers and replace with backward compatible config fetching.

We should do it in a follow-up PR to limit the scope of this PR.